### ISSUE TITLE
Passwords with special characters, white space,... ?

### ISSUE THREAD

**Issue by github-user-1** [davecb/ssmtp#2]
In case someone is listening here - I can't prove this is the root cause, but after quite some digging I'm convinced my issues comes from a complex password (including special characters, white spaces, etc.). Is there something special to consider in the config-file for such kind of passwords (brackets, quotes, escaping, ...)?

**Comment by davecb** [comment]
I think you're in the wrong issue-tracker: ssmtp doesn't have passwords, just a config file like

root=davecb # The person who gets root's mail (also daemons', etc)
mailhub=ccs.yorku.ca # The place where the mail goes.
rewriteDomain=ccs.yorku.ca # The place to use in From: lines.
hostname=draco.ccs.yorku.ca # Optional, if you don't use FQDNs as hostnames.

**Comment by github-user-1** [comment]
Right, the config-file being part of ssmtp (at least in my setup, debian) is located at: /etc/ssmtp/ssmtp.conf.

It contains information just like above on the mailhub, etc. but also the required login credentials for it. E.g. something like (following several guides):
```
...
mailhub=examplemail.org:587
UseSTARTTLS=YES
AuthUser=myuserid
AuthPass=My Pass Wd #\ ! 
AuthLogin=Yes
...
```
Sorry, in case the above is not part of the ssmtp config, but I just thought so and was wondering about how exactly the password should be given if it's more complex like e.g. above (since I think the config might not fully catch it).

**Comment by davecb** [comment]
Ah, you have a different implementation, at https://anonscm.debian.org/gitweb/?p=ssmtp/ssmtp.git
Thanks for the issue, I can add a pointer to the newer version from here now.
